Src/AutoMoq/MockVirtualMethodsCommand.cs

```diff
@@ -115,6 +116,14 @@ private static IEnumerable<MethodInfo> GetConfigurableMethods(Type type)
                               ? type.GetInterfaceMethods()
                               : type.GetMethods();
 
+            // Skip properties that have both getters and setters to not interfere
+            // with other post-processors in the chain which initialize them."
+            var getSetMethods = type.GetProperties()
+                                    .Where(p => p.GetGetMethod() != null &&
+                                                p.GetSetMethod() != null)
+                                    .SelectMany(p => new[] {p.GetGetMethod(), p.GetSetMethod()});
```

Comment:
You're right. Selecting `GetSetMethod()` was not necessary. Was just to be more clear about the intention, but I agree that its simpler without it (and probably performs a bit better).

I've also merged the improvements with the original commits. Thought it would be more clear what I improved, but if you don't think its necessary I'm fine with merging them.